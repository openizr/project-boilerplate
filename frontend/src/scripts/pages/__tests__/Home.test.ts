/**
 * @jest-environment jsdom
 */

import { render } from '@testing-library/svelte';
import Home from 'scripts/pages/Home.svelte';

describe('pages/Home', () => {
  beforeEach(() => {
    jest.clearAllMocks();
  });

  test('renders correctly - basic', () => {
    const { container } = render(Home, { props: { locale: { LABEL_TEST: 'Test' } } });
    expect(container.firstChild).toMatchSnapshot();
  });
});
