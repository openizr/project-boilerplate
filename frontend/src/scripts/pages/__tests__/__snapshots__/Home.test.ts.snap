// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/Home renders correctly - basic 1`] = `
<div>
  <p>
    <img
      alt="test"
      class="svelte-1int8dr"
    />
     
    <img
      alt="test"
      class="svelte-1int8dr"
    />
     
    Test
  </p>
</div>
`;
